gadget-container/gadgettracermanager/main.go

```diff
@@ -39,6 +39,8 @@ import (
 	gadgetservice "github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgettracermanager"
 	pb "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgettracermanager/api"
+
+	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/bpftrace"
```

Comment:
Right now other gadgets are working because they are included in the gadget collection package. For this one we need a explicit include to make it work. I added a comment.